docs/callbacks.html

```diff
@@ -652,7 +652,7 @@ <h3 id="SaveModelCallback"><a href="/callbacks.tracker.html#SaveModelCallback"><
 <div class="output_area">
 
 <div class="output_subarea output_stream output_stdout output_text">
-<pre>bestmodel_1.pth  bestmodel_2.pth  bestmodel_3.pth
+<pre>bestmodel_1.pth  bestmodel_2.pth  bestmodel_3.pth
 </pre>
 </div>
 </div>
```

Comment:
this one doesn't belong here, correct? What happens if hit [delete] - will it just remove it from the PR, but leave the file in place?

Reply:
Yes, I don't know how it got there. I'll clean up the commit.